README.md

```diff
@@ -12,38 +11,35 @@ Before starting the practical, make sure you have the following ready, if not, p
+5. Provide a name for your questionnaire, try to use a name that can distinguish your record from other trainees, for example include your user name in the questionnaire name. In addition, you may share your questionnaire with others. You may allow either DSW tool users or anyone with a link to view/edit your answers. Clicking `Save` will create your questionnaire. <br/><br/><br/> ![Alt](images/project_name.png)<br/><br/><br/>
 
 ## Exercice 3: Preview (empty) report from your questionnaire
 
- * To have a preview report from your questionnaire: on the navigation menu, click on the `Settings` button, select the required template (FNR Template) and the `HTML` default format . Then click on `Preview` to go to the overview of the document and see the different sections of the FNR template (empty) at the moment.<br/><br/><br/>![Alt](images/settings.png)<br/><br/><br/>
+* To have a preview report from your questionnaire: on the navigation menu, click on the `Settings` button, select the required template (FNR Template) and the `HTML` default format . Then click on `Preview` to go to the overview of the document and see the different sections of the FNR template (empty) at the moment.<br/><br/><br/>![Alt](images/settings.png)<br/><br/><br/>
 
 ## Exercice 4: Answer questions in the questionnaire
 
-1. Go back to the your questionnaire which is ready to be filled. Under most of the questions you will find hints for the questions. <br/><br/><br/> ![Alt](images/fill1.png)<br/><br/><br/>
+1. Go back to the your questionnaire which is ready to be filled. Under most of the questions you will find hints related to the training scenario. <br/><br/><br/> ![Alt](images/fill1.png)<br/><br/><br/>
 2. The DSW provides a filtering feature that allows you to see questions only relevant to a particular phase of a research project. From the `Current Phase` list select `Before Submitting the DMP`. This will allow you to view and answers all questions for our practical. <br/><br/><br/> ![Alt](images/fill2.png)<br/><br/><br/>
 
 When filling in the questionnaire pay attention to the following:
 
-
- - for some questions you may provide more than one answer. If you want to add a new answer click the `Add` button at the end of your last answer. <br/><br/><br/>![Alt](images/add_answer.png)<br/><br/><br/>
+- for some questions you may provide more than one answer. If you want to add a new answer click the `Add` button at the end of your last answer. <br/><br/><br/>![Alt](images/add_answer.png)<br/><br/><br/>
 
 ## Exercise 5: Export (filled) report from your questionnaire
```

Comment:
I would name this one "Generate report of your filled questionnaire"

Reply:
Agreed.